libs/product/testing/src/factories/magento/simple/simple.factory.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
```

Comment:
Let's remove this from this PR, its outside scope.

Reply:
removed